tools/mtmd/clip.cpp

```diff
@@ -4606,6 +4634,16 @@ int clip_n_output_tokens(const struct clip_ctx * ctx, struct clip_image_f32 * im
                     n_patches /= 2;
                 }
             } break;
+        case PROJECTOR_TYPE_GLMA:
+            {
+                n_patches = img->nx;
+                // whisper downscales input token by half after conv1d
+                n_patches /= 2;
+                // reshape by merge_factor
+                n_patches /= 4;
```

Comment:
```suggestion
                n_patches /= n_merge;
```

and you also need to set `hparams.n_merge = 4` upon loading hparams, see `load_hparams()` function